### ISSUE TITLE
gitflow workflow: the best workflow for release-based projects

### ISSUE THREAD

**Issue by artic92** [artic92/howtos#18]
1. A develop branch is created from master
2. A release branch is created from develop
3. Feature branches are created from develop
4. When a feature is complete it is merged into the develop branch
5. When the release branch is done it is merged into develop and master
6. If an issue in master is detected a hotfix branch is created from master
7. Once the hotfix is complete it is merged to both develop and master

source: <https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow>
